tests/core/test_domain.py

```diff
@@ -501,3 +500,28 @@ def test_load_on_invalid_domain():
     # Currently just deprecated
     # with pytest.raises(InvalidDomain):
     #     Domain.load("data/test_domains/missing_text_for_templates.yml")
+
+
+def test_clean_domain():
+    domain_path = "data/test_domains/default_unfeaturized_entities.yml"
```

Comment:
Should we add an intent without a dict value? E.g.
```
intents:
  - mood
```

Reply:
done